core/src/test/java/com/clubobsidian/dynamicgui/core/test/function/ConditionFunctionTest.java

```diff
@@ -0,0 +1,185 @@
+package com.clubobsidian.dynamicgui.core.test.function;
+
+import com.clubobsidian.dynamicgui.api.entity.PlayerWrapper;
+import com.clubobsidian.dynamicgui.core.function.condition.*;
+import com.clubobsidian.dynamicgui.core.test.mock.test.FactoryTest;
+import com.ezylang.evalex.EvaluationException;
+import com.ezylang.evalex.Expression;
+import com.ezylang.evalex.config.ExpressionConfiguration;
+import com.ezylang.evalex.parser.ParseException;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ConditionFunctionTest extends FactoryTest {
+    ExpressionConfiguration config = ExpressionConfiguration.defaultConfiguration().withAdditionalFunctions(
```

Comment:
Remove this and wrap each test in a condition function instead. So code will work something like this
```java
ConditionFunction function = new ConditionFunction();
function.setData("expressionString");
assertTrue/assertFalse(function.function(this.getFactory().createPlayer());
```

Reply:
Will do.

Reply:
Addressed in latest commit.